native/core/src/events/calendar.rs

```diff
@@ -22,34 +22,18 @@ pub enum EventLocation {
         #[serde(default, skip_serializing_if = "Option::is_none")]
         name: Option<String>,
 
-        /// further description to this location
-        #[serde(default, skip_serializing_if = "Option::is_none")]
-        description: Option<TextMessageEventContent>,
-
```

Comment:
Sorry, this is needed (for future usage). Consider that the address of a physical location might need some extra information, e.g. some directions within a complex that can't be easily done in a one-liner...